lib/libutils/isoc/bget_malloc.c

```diff
@@ -691,7 +705,7 @@ static void assert_header(struct mdbg_hdr *hdr)
 	assert(*mdbg_get_footer(hdr) == MDBG_FOOTER_MAGIC);
 }
 
-void mdbg_free(void *ptr)
+static void mdbg_free(void *ptr)
```

Comment:
@jenswi-linaro ah, yes. So no `.malloc` or `.calloc` etc. either (but those are less likely to be used I suppose).